app/controllers/admin/events_controller.rb

```diff
@@ -3,7 +3,10 @@ class EventsController < BaseController
     helper_method :event, :events
 
     def new
-      @event = Event.new
+      @event = Event.new(
+        started_at: date_with_specified_hour(Event::DEFAULT_STARTED_AT_HOURS),
```

Comment:
this looks like business logic in controller. Bad.
Try extract this logic to service like `@event = Event::New.call` and let service to rule logic